godaddy-email-marketing.php

```diff
@@ -71,11 +71,13 @@ public static function instance() {
 	 * @codeCoverageIgnore
 	 */
 	private function setup_actions() {
-		add_action( 'plugins_loaded', array( $this, 'i18n' ) );
-		add_action( 'init',           array( $this, 'init' ) );
-		add_action( 'widgets_init',   array( $this, 'register_widget' ) );
-		add_action( 'init',           array( $this, 'register_shortcode' ), 20 );
-		add_action( 'admin_notices',  array( $this, 'action_admin_notices' ) );
+		add_action( 'plugins_loaded',                  array( $this, 'i18n' ) );
+		add_action( 'init',                            array( $this, 'init' ) );
+		add_action( 'widgets_init',                    array( $this, 'register_widget' ) );
+		add_action( 'init',                            array( $this, 'register_shortcode' ), 20 );
+		add_Action( 'admin_enqueue_scripts',           array( $this, 'register_admin_scripts' ) );
+		add_action( 'admin_notices',                   array( $this, 'action_admin_notices' ) );
+		add_action( 'wp_ajax_dismiss_gem_notice',      array( $this, 'delete_wpem_gem_notice_option' ) );
```

Comment:
@EvanHerman We can reduce the spacing between the filter name and the array. Only 1 space between the comman of the biggest filter name.